lib/operations/operation.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+const Aspect = {
+  SKIP_SESSION: Symbol('SKIP_SESSION')
+};
+
+class OperationBase {
+  constructor(options, aspects) {
+    this.options = options || {};
+    this.aspects = aspects || new Set([]);
```

Comment:
if we are using `defineAspects` then we shouldn't allow them to be set in the constructor like this.